### ISSUE TITLE
Patch Panels: can not select same customer / port details after a reset

### ISSUE THREAD

**Issue by barryo** [inex/IXP-Manager#350]
On an already allocated patch panel port, if I edit and reset the customer / switch port details, I cannot set them back the way they were.

To reproduce:

1. Find an already allocated patch panel port (with a customer and switch port)
2. Edit it (Action dropdown -> Edit)
3. Click either reset button
4. If you pick the same customer, you cannot find the original switch/switch port
5. If you instead go to select the switch before the customer, you cannot find the original switch/switch port

In other words: the customer's original switch port is not available in the dropdowns.
